src/test/pages/SetAssignments.test.tsx

```diff
@@ -160,10 +160,7 @@ describe("SetAssignments", () => {
     const allAssignments = currentAssignments.concat(pendingAssignments);
     expect(allAssignments).toHaveLength(1);
     expect(allAssignments[0].textContent).toContain(mockActiveGroups[0].groupName);
-
-    // Click button
-    const assignButton = within(modal).getByRole("button", { name: "Assign to group" });
-    await userEvent.click(assignButton);
+    await clickButton("Assign to group");
```

Comment:
Same problem here. Simply adding an empty line above this one would be a good way to separate the two blocks.